packages/fuels/test/utils/mockCheckForUpdates.ts

```diff
@@ -0,0 +1,7 @@
+import * as checkForUpdatesMod from '../../src/cli/utils/checkForAndDisplayUpdates';
+
+export const mockCheckForUpdates = () => {
```

Comment:
Why do we always mock a successful resolution?

Reply:
Because as of now this helper for the mock is only used in places where we always expect this functionality to succeed or not run.